bookingsRoutes.js

```diff
@@ -0,0 +1,131 @@
+const express = require("express");
+const bookings = require("./bookings.json");
+const fs = require("fs").promises;
+
+const bookingsRouter = express.Router();
+
+
+// ----------- Getting All -----------
+const getAllBookings = (req, res) => {
+  fs.readFile("./bookings.json", "utf8")
+    .then(bookings => {
+      bookings = JSON.parse(bookings);
+      res.status(200).json({ data: bookings });
+      return;
```

Comment:
Some error handling here might be nice (e.g. `.catch` or converting to `async/await` and using a `try/catch` block)